
 
记录时间：2017-05-19 16:24:38,721 
线程ID:[5] 
日志级别：  ERROR 
出错类：HCQ2WebAPI_Logic.DExceptionFilterAttribute property: [(null)] - 
错误描述：
.捕获异常信息：
IP：::1
浏览器：Unknown
浏览器版本：0
操作系统：Unknown
错误信息：
错误页面：http://localhost/HCQ2WebAPI/api/DebtWage/GetDebtTime
错误信息：未将对象引用设置到对象的实例。
错误源：HCQ2_Common
异常方法：System.Object GetSessionValue(System.String)
堆栈信息：   在 HCQ2_Common.Bean.SessionHelper.GetSessionValue(String SESSION_NAME) 位置 G:\NET项目\TaoCode\HCQ2\HCQ2_Common\Bean\SessionHelper.cs:行号 57
   在 HCQ2UI_Helper.Session.SysPermissSession.get_PerminssUnitData() 位置 G:\NET项目\TaoCode\HCQ2\HCQ2UI_Helper\Session\SysPermissSession.cs:行号 136
   在 HCQ2_DAL_MSSQL.View_QXTJDAL.GetDebtTime(DebtSelModel model) 位置 G:\NET项目\TaoCode\HCQ2\HCQ2_DAL\ExtensionDAL\View_QXTJDAL.cs:行号 129
   在 HCQ2_BLL.View_QXTJBLL.GetDebtTime(DebtSelModel model) 位置 G:\NET项目\TaoCode\HCQ2\HCQ2_BLL\ExtensionBLL\View_QXTJBLL.cs:行号 258
   在 HCQ2WebAPI_Logic.APPController.DebtWageController.GetDebtTime(DebtSelModel model) 位置 G:\NET项目\TaoCode\HCQ2\HCQ2WebAPI_Logic\APPController\DebtWageController.cs:行号 31
   在 System.Web.Http.Controllers.ReflectedHttpActionDescriptor.ActionExecutor.<>c__DisplayClass10.<GetExecutor>b__9(Object instance, Object[] methodParameters)
   在 System.Web.Http.Controllers.ReflectedHttpActionDescriptor.ExecuteAsync(HttpControllerContext controllerContext, IDictionary`2 arguments, CancellationToken cancellationToken)
--- 引发异常的上一位置中堆栈跟踪的末尾 ---
   在 System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   在 System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   在 System.Web.Http.Controllers.ApiControllerActionInvoker.<InvokeActionAsyncCore>d__0.MoveNext()
--- 引发异常的上一位置中堆栈跟踪的末尾 ---
   在 System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   在 System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   在 System.Web.Http.Controllers.ActionFilterResult.<ExecuteAsync>d__2.MoveNext()
--- 引发异常的上一位置中堆栈跟踪的末尾 ---
   在 System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   在 System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   在 System.Web.Http.Filters.AuthorizationFilterAttribute.<ExecuteAuthorizationFilterAsyncCore>d__2.MoveNext()
--- 引发异常的上一位置中堆栈跟踪的末尾 ---
   在 System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   在 System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   在 System.Web.Http.Controllers.AuthenticationFilterResult.<ExecuteAsync>d__0.MoveNext()
--- 引发异常的上一位置中堆栈跟踪的末尾 ---
   在 System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   在 System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   在 System.Web.Http.Controllers.ExceptionFilterResult.<ExecuteAsync>d__0.MoveNext()

 

 
记录时间：2017-05-19 16:26:36,010 
线程ID:[18] 
日志级别：  ERROR 
出错类：HCQ2WebAPI_Logic.DExceptionFilterAttribute property: [(null)] - 
错误描述：
.捕获异常信息：
IP：::1
浏览器：Unknown
浏览器版本：0
操作系统：Unknown
错误信息：
错误页面：http://localhost/HCQ2WebAPI/api/DebtWage/GetDebtTime
错误信息：未将对象引用设置到对象的实例。
错误源：HCQ2_Common
异常方法：System.Object GetSessionValue(System.String)
堆栈信息：   在 HCQ2_Common.Bean.SessionHelper.GetSessionValue(String SESSION_NAME) 位置 G:\NET项目\TaoCode\HCQ2\HCQ2_Common\Bean\SessionHelper.cs:行号 57
   在 HCQ2UI_Helper.Session.SysPermissSession.get_PerminssUnitData() 位置 G:\NET项目\TaoCode\HCQ2\HCQ2UI_Helper\Session\SysPermissSession.cs:行号 136
   在 HCQ2_DAL_MSSQL.View_QXTJDAL.GetDebtTime(DebtSelModel model) 位置 G:\NET项目\TaoCode\HCQ2\HCQ2_DAL\ExtensionDAL\View_QXTJDAL.cs:行号 129
   在 HCQ2_BLL.View_QXTJBLL.GetDebtTime(DebtSelModel model) 位置 G:\NET项目\TaoCode\HCQ2\HCQ2_BLL\ExtensionBLL\View_QXTJBLL.cs:行号 258
   在 HCQ2WebAPI_Logic.APPController.DebtWageController.GetDebtTime(DebtSelModel model) 位置 G:\NET项目\TaoCode\HCQ2\HCQ2WebAPI_Logic\APPController\DebtWageController.cs:行号 31
   在 System.Web.Http.Controllers.ReflectedHttpActionDescriptor.ActionExecutor.<>c__DisplayClass10.<GetExecutor>b__9(Object instance, Object[] methodParameters)
   在 System.Web.Http.Controllers.ReflectedHttpActionDescriptor.ExecuteAsync(HttpControllerContext controllerContext, IDictionary`2 arguments, CancellationToken cancellationToken)
--- 引发异常的上一位置中堆栈跟踪的末尾 ---
   在 System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   在 System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   在 System.Web.Http.Controllers.ApiControllerActionInvoker.<InvokeActionAsyncCore>d__0.MoveNext()
--- 引发异常的上一位置中堆栈跟踪的末尾 ---
   在 System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   在 System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   在 System.Web.Http.Controllers.ActionFilterResult.<ExecuteAsync>d__2.MoveNext()
--- 引发异常的上一位置中堆栈跟踪的末尾 ---
   在 System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   在 System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   在 System.Web.Http.Filters.AuthorizationFilterAttribute.<ExecuteAuthorizationFilterAsyncCore>d__2.MoveNext()
--- 引发异常的上一位置中堆栈跟踪的末尾 ---
   在 System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   在 System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   在 System.Web.Http.Controllers.AuthenticationFilterResult.<ExecuteAsync>d__0.MoveNext()
--- 引发异常的上一位置中堆栈跟踪的末尾 ---
   在 System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   在 System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   在 System.Web.Http.Controllers.ExceptionFilterResult.<ExecuteAsync>d__0.MoveNext()

 

 
记录时间：2017-05-19 16:30:31,121 
线程ID:[18] 
日志级别：  ERROR 
出错类：HCQ2WebAPI_Logic.DExceptionFilterAttribute property: [(null)] - 
错误描述：
.捕获异常信息：
IP：::1
浏览器：Unknown
浏览器版本：0
操作系统：Unknown
错误信息：
错误页面：http://localhost/HCQ2WebAPI/api/DebtWage/GetDebtTime
错误信息：未将对象引用设置到对象的实例。
错误源：HCQ2_Common
异常方法：System.Object GetSessionValue(System.String)
堆栈信息：   在 HCQ2_Common.Bean.SessionHelper.GetSessionValue(String SESSION_NAME) 位置 G:\NET项目\TaoCode\HCQ2\HCQ2_Common\Bean\SessionHelper.cs:行号 57
   在 HCQ2UI_Helper.Session.SysPermissSession.get_PerminssUnitData() 位置 G:\NET项目\TaoCode\HCQ2\HCQ2UI_Helper\Session\SysPermissSession.cs:行号 136
   在 HCQ2_DAL_MSSQL.View_QXTJDAL.GetDebtTime(DebtSelModel model) 位置 G:\NET项目\TaoCode\HCQ2\HCQ2_DAL\ExtensionDAL\View_QXTJDAL.cs:行号 129
   在 HCQ2_BLL.View_QXTJBLL.GetDebtTime(DebtSelModel model) 位置 G:\NET项目\TaoCode\HCQ2\HCQ2_BLL\ExtensionBLL\View_QXTJBLL.cs:行号 258
   在 HCQ2WebAPI_Logic.APPController.DebtWageController.GetDebtTime(DebtSelModel model) 位置 G:\NET项目\TaoCode\HCQ2\HCQ2WebAPI_Logic\APPController\DebtWageController.cs:行号 31
   在 System.Web.Http.Controllers.ReflectedHttpActionDescriptor.ActionExecutor.<>c__DisplayClass10.<GetExecutor>b__9(Object instance, Object[] methodParameters)
   在 System.Web.Http.Controllers.ReflectedHttpActionDescriptor.ExecuteAsync(HttpControllerContext controllerContext, IDictionary`2 arguments, CancellationToken cancellationToken)
--- 引发异常的上一位置中堆栈跟踪的末尾 ---
   在 System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   在 System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   在 System.Web.Http.Controllers.ApiControllerActionInvoker.<InvokeActionAsyncCore>d__0.MoveNext()
--- 引发异常的上一位置中堆栈跟踪的末尾 ---
   在 System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   在 System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   在 System.Web.Http.Controllers.ActionFilterResult.<ExecuteAsync>d__2.MoveNext()
--- 引发异常的上一位置中堆栈跟踪的末尾 ---
   在 System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   在 System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   在 System.Web.Http.Filters.AuthorizationFilterAttribute.<ExecuteAuthorizationFilterAsyncCore>d__2.MoveNext()
--- 引发异常的上一位置中堆栈跟踪的末尾 ---
   在 System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   在 System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   在 System.Web.Http.Controllers.AuthenticationFilterResult.<ExecuteAsync>d__0.MoveNext()
--- 引发异常的上一位置中堆栈跟踪的末尾 ---
   在 System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   在 System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   在 System.Web.Http.Controllers.ExceptionFilterResult.<ExecuteAsync>d__0.MoveNext()

 

 
记录时间：2017-05-19 16:31:20,299 
线程ID:[18] 
日志级别：  ERROR 
出错类：HCQ2WebAPI_Logic.DExceptionFilterAttribute property: [(null)] - 
错误描述：
.捕获异常信息：
IP：::1
浏览器：Unknown
浏览器版本：0
操作系统：Unknown
错误信息：
错误页面：http://localhost/HCQ2WebAPI/api/DebtWage/GetDebtTime
错误信息：未将对象引用设置到对象的实例。
错误源：HCQ2_Common
异常方法：System.Object GetSessionValue(System.String)
堆栈信息：   在 HCQ2_Common.Bean.SessionHelper.GetSessionValue(String SESSION_NAME) 位置 G:\NET项目\TaoCode\HCQ2\HCQ2_Common\Bean\SessionHelper.cs:行号 57
   在 HCQ2UI_Helper.Session.SysPermissSession.get_PerminssUnitData() 位置 G:\NET项目\TaoCode\HCQ2\HCQ2UI_Helper\Session\SysPermissSession.cs:行号 136
   在 HCQ2_DAL_MSSQL.View_QXTJDAL.GetDebtTime(DebtSelModel model) 位置 G:\NET项目\TaoCode\HCQ2\HCQ2_DAL\ExtensionDAL\View_QXTJDAL.cs:行号 129
   在 HCQ2_BLL.View_QXTJBLL.GetDebtTime(DebtSelModel model) 位置 G:\NET项目\TaoCode\HCQ2\HCQ2_BLL\ExtensionBLL\View_QXTJBLL.cs:行号 258
   在 HCQ2WebAPI_Logic.APPController.DebtWageController.GetDebtTime(DebtSelModel model) 位置 G:\NET项目\TaoCode\HCQ2\HCQ2WebAPI_Logic\APPController\DebtWageController.cs:行号 31
   在 System.Web.Http.Controllers.ReflectedHttpActionDescriptor.ActionExecutor.<>c__DisplayClass10.<GetExecutor>b__9(Object instance, Object[] methodParameters)
   在 System.Web.Http.Controllers.ReflectedHttpActionDescriptor.ExecuteAsync(HttpControllerContext controllerContext, IDictionary`2 arguments, CancellationToken cancellationToken)
--- 引发异常的上一位置中堆栈跟踪的末尾 ---
   在 System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   在 System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   在 System.Web.Http.Controllers.ApiControllerActionInvoker.<InvokeActionAsyncCore>d__0.MoveNext()
--- 引发异常的上一位置中堆栈跟踪的末尾 ---
   在 System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   在 System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   在 System.Web.Http.Controllers.ActionFilterResult.<ExecuteAsync>d__2.MoveNext()
--- 引发异常的上一位置中堆栈跟踪的末尾 ---
   在 System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   在 System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   在 System.Web.Http.Filters.AuthorizationFilterAttribute.<ExecuteAuthorizationFilterAsyncCore>d__2.MoveNext()
--- 引发异常的上一位置中堆栈跟踪的末尾 ---
   在 System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   在 System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   在 System.Web.Http.Controllers.AuthenticationFilterResult.<ExecuteAsync>d__0.MoveNext()
--- 引发异常的上一位置中堆栈跟踪的末尾 ---
   在 System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   在 System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   在 System.Web.Http.Controllers.ExceptionFilterResult.<ExecuteAsync>d__0.MoveNext()

 

 
记录时间：2017-05-19 16:36:26,846 
线程ID:[47] 
日志级别：  ERROR 
出错类：HCQ2WebAPI_Logic.DExceptionFilterAttribute property: [(null)] - 
错误描述：
.捕获异常信息：
IP：::1
浏览器：Unknown
浏览器版本：0
操作系统：Unknown
错误信息：
错误页面：http://localhost/HCQ2WebAPI/api/DebtWage/GetDebtTime
错误信息：未将对象引用设置到对象的实例。
错误源：HCQ2_Common
异常方法：System.Object GetSessionValue(System.String)
堆栈信息：   在 HCQ2_Common.Bean.SessionHelper.GetSessionValue(String SESSION_NAME) 位置 G:\NET项目\TaoCode\HCQ2\HCQ2_Common\Bean\SessionHelper.cs:行号 57
   在 HCQ2UI_Helper.Session.SysPermissSession.get_PerminssUnitData() 位置 G:\NET项目\TaoCode\HCQ2\HCQ2UI_Helper\Session\SysPermissSession.cs:行号 136
   在 HCQ2_BLL.View_QXTJBLL.GetDebtTime(DebtSelModel model) 位置 G:\NET项目\TaoCode\HCQ2\HCQ2_BLL\ExtensionBLL\View_QXTJBLL.cs:行号 258
   在 HCQ2WebAPI_Logic.APPController.DebtWageController.GetDebtTime(DebtSelModel model) 位置 G:\NET项目\TaoCode\HCQ2\HCQ2WebAPI_Logic\APPController\DebtWageController.cs:行号 31
   在 System.Web.Http.Controllers.ReflectedHttpActionDescriptor.ActionExecutor.<>c__DisplayClass10.<GetExecutor>b__9(Object instance, Object[] methodParameters)
   在 System.Web.Http.Controllers.ReflectedHttpActionDescriptor.ExecuteAsync(HttpControllerContext controllerContext, IDictionary`2 arguments, CancellationToken cancellationToken)
--- 引发异常的上一位置中堆栈跟踪的末尾 ---
   在 System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   在 System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   在 System.Web.Http.Controllers.ApiControllerActionInvoker.<InvokeActionAsyncCore>d__0.MoveNext()
--- 引发异常的上一位置中堆栈跟踪的末尾 ---
   在 System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   在 System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   在 System.Web.Http.Controllers.ActionFilterResult.<ExecuteAsync>d__2.MoveNext()
--- 引发异常的上一位置中堆栈跟踪的末尾 ---
   在 System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   在 System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   在 System.Web.Http.Filters.AuthorizationFilterAttribute.<ExecuteAuthorizationFilterAsyncCore>d__2.MoveNext()
--- 引发异常的上一位置中堆栈跟踪的末尾 ---
   在 System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   在 System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   在 System.Web.Http.Controllers.AuthenticationFilterResult.<ExecuteAsync>d__0.MoveNext()
--- 引发异常的上一位置中堆栈跟踪的末尾 ---
   在 System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   在 System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   在 System.Web.Http.Controllers.ExceptionFilterResult.<ExecuteAsync>d__0.MoveNext()

 

 
记录时间：2017-05-19 17:00:20,411 
线程ID:[48] 
日志级别：  ERROR 
出错类：HCQ2WebAPI_Logic.DExceptionFilterAttribute property: [(null)] - 
错误描述：
.捕获异常信息：
IP：::1
浏览器：Unknown
浏览器版本：0
操作系统：Unknown
错误信息：
错误页面：http://localhost/HCQ2WebAPI/api/DebtWage/GetDebtMonthMoney
错误信息：从字符串转换日期和/或时间时，转换失败。
错误源：.Net SqlClient Data Provider
异常方法：Void OnError(System.Data.SqlClient.SqlException, Boolean, System.Action`1[System.Action])
堆栈信息：   在 System.Data.SqlClient.SqlConnection.OnError(SqlException exception, Boolean breakConnection, Action`1 wrapCloseInAction)
   在 System.Data.SqlClient.TdsParser.ThrowExceptionAndWarning(TdsParserStateObject stateObj, Boolean callerHasConnectionLock, Boolean asyncClose)
   在 System.Data.SqlClient.TdsParser.TryRun(RunBehavior runBehavior, SqlCommand cmdHandler, SqlDataReader dataStream, BulkCopySimpleResultSet bulkCopyHandler, TdsParserStateObject stateObj, Boolean& dataReady)
   在 System.Data.SqlClient.SqlDataReader.TryHasMoreRows(Boolean& moreRows)
   在 System.Data.SqlClient.SqlDataReader.TryReadInternal(Boolean setTimeout, Boolean& more)
   在 System.Data.SqlClient.SqlDataReader.Read()
   在 System.Data.Common.DataAdapter.FillLoadDataRow(SchemaMapping mapping)
   在 System.Data.Common.DataAdapter.FillFromReader(DataSet dataset, DataTable datatable, String srcTable, DataReaderContainer dataReader, Int32 startRecord, Int32 maxRecords, DataColumn parentChapterColumn, Object parentChapterValue)
   在 System.Data.Common.DataAdapter.Fill(DataTable[] dataTables, IDataReader dataReader, Int32 startRecord, Int32 maxRecords)
   在 System.Data.Common.DbDataAdapter.FillInternal(DataSet dataset, DataTable[] datatables, Int32 startRecord, Int32 maxRecords, String srcTable, IDbCommand command, CommandBehavior behavior)
   在 System.Data.Common.DbDataAdapter.Fill(DataTable[] dataTables, Int32 startRecord, Int32 maxRecords, IDbCommand command, CommandBehavior behavior)
   在 System.Data.Common.DbDataAdapter.Fill(DataTable dataTable)
   在 HCQ2_Common.SQL.SqlHelper.ExecuteDataTable(String sql, CommandType cmdType, SqlParameter[] psm) 位置 G:\NET项目\TaoCode\HCQ2\HCQ2_Common\SQL\SqlHelper.cs:行号 176
   在 HCQ2_DAL_MSSQL.View_QXTJDAL.GetDebtChartMonthDataByOrg(DebtSelGrantModel model) 位置 G:\NET项目\TaoCode\HCQ2\HCQ2_DAL\ExtensionDAL\View_QXTJDAL.cs:行号 201
   在 HCQ2_BLL.View_QXTJBLL.GetDebtChartMonthDataByOrg(DebtSelGrantModel model) 位置 G:\NET项目\TaoCode\HCQ2\HCQ2_BLL\ExtensionBLL\View_QXTJBLL.cs:行号 280
   在 HCQ2WebAPI_Logic.APPController.DebtWageController.GetDebtMonthMoney(DebtSelGrantModel model) 位置 G:\NET项目\TaoCode\HCQ2\HCQ2WebAPI_Logic\APPController\DebtWageController.cs:行号 85
   在 System.Web.Http.Controllers.ReflectedHttpActionDescriptor.ActionExecutor.<>c__DisplayClass10.<GetExecutor>b__9(Object instance, Object[] methodParameters)
   在 System.Web.Http.Controllers.ReflectedHttpActionDescriptor.ExecuteAsync(HttpControllerContext controllerContext, IDictionary`2 arguments, CancellationToken cancellationToken)
--- 引发异常的上一位置中堆栈跟踪的末尾 ---
   在 System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   在 System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   在 System.Web.Http.Controllers.ApiControllerActionInvoker.<InvokeActionAsyncCore>d__0.MoveNext()
--- 引发异常的上一位置中堆栈跟踪的末尾 ---
   在 System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   在 System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   在 System.Web.Http.Controllers.ActionFilterResult.<ExecuteAsync>d__2.MoveNext()
--- 引发异常的上一位置中堆栈跟踪的末尾 ---
   在 System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   在 System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   在 System.Web.Http.Filters.AuthorizationFilterAttribute.<ExecuteAuthorizationFilterAsyncCore>d__2.MoveNext()
--- 引发异常的上一位置中堆栈跟踪的末尾 ---
   在 System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   在 System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   在 System.Web.Http.Controllers.AuthenticationFilterResult.<ExecuteAsync>d__0.MoveNext()
--- 引发异常的上一位置中堆栈跟踪的末尾 ---
   在 System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   在 System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   在 System.Web.Http.Controllers.ExceptionFilterResult.<ExecuteAsync>d__0.MoveNext()

 

 
记录时间：2017-05-19 17:01:15,822 
线程ID:[36] 
日志级别：  ERROR 
出错类：HCQ2WebAPI_Logic.DExceptionFilterAttribute property: [(null)] - 
错误描述：
.捕获异常信息：
IP：::1
浏览器：Unknown
浏览器版本：0
操作系统：Unknown
错误信息：
错误页面：http://localhost/HCQ2WebAPI/api/DebtWage/GetDebtMonthMoney
错误信息：从字符串转换日期和/或时间时，转换失败。
错误源：.Net SqlClient Data Provider
异常方法：Void OnError(System.Data.SqlClient.SqlException, Boolean, System.Action`1[System.Action])
堆栈信息：   在 System.Data.SqlClient.SqlConnection.OnError(SqlException exception, Boolean breakConnection, Action`1 wrapCloseInAction)
   在 System.Data.SqlClient.TdsParser.ThrowExceptionAndWarning(TdsParserStateObject stateObj, Boolean callerHasConnectionLock, Boolean asyncClose)
   在 System.Data.SqlClient.TdsParser.TryRun(RunBehavior runBehavior, SqlCommand cmdHandler, SqlDataReader dataStream, BulkCopySimpleResultSet bulkCopyHandler, TdsParserStateObject stateObj, Boolean& dataReady)
   在 System.Data.SqlClient.SqlDataReader.TryHasMoreRows(Boolean& moreRows)
   在 System.Data.SqlClient.SqlDataReader.TryReadInternal(Boolean setTimeout, Boolean& more)
   在 System.Data.SqlClient.SqlDataReader.Read()
   在 System.Data.Common.DataAdapter.FillLoadDataRow(SchemaMapping mapping)
   在 System.Data.Common.DataAdapter.FillFromReader(DataSet dataset, DataTable datatable, String srcTable, DataReaderContainer dataReader, Int32 startRecord, Int32 maxRecords, DataColumn parentChapterColumn, Object parentChapterValue)
   在 System.Data.Common.DataAdapter.Fill(DataTable[] dataTables, IDataReader dataReader, Int32 startRecord, Int32 maxRecords)
   在 System.Data.Common.DbDataAdapter.FillInternal(DataSet dataset, DataTable[] datatables, Int32 startRecord, Int32 maxRecords, String srcTable, IDbCommand command, CommandBehavior behavior)
   在 System.Data.Common.DbDataAdapter.Fill(DataTable[] dataTables, Int32 startRecord, Int32 maxRecords, IDbCommand command, CommandBehavior behavior)
   在 System.Data.Common.DbDataAdapter.Fill(DataTable dataTable)
   在 HCQ2_Common.SQL.SqlHelper.ExecuteDataTable(String sql, CommandType cmdType, SqlParameter[] psm) 位置 G:\NET项目\TaoCode\HCQ2\HCQ2_Common\SQL\SqlHelper.cs:行号 176
   在 HCQ2_DAL_MSSQL.View_QXTJDAL.GetDebtChartMonthDataByOrg(DebtSelGrantModel model) 位置 G:\NET项目\TaoCode\HCQ2\HCQ2_DAL\ExtensionDAL\View_QXTJDAL.cs:行号 201
   在 HCQ2_BLL.View_QXTJBLL.GetDebtChartMonthDataByOrg(DebtSelGrantModel model) 位置 G:\NET项目\TaoCode\HCQ2\HCQ2_BLL\ExtensionBLL\View_QXTJBLL.cs:行号 280
   在 HCQ2WebAPI_Logic.APPController.DebtWageController.GetDebtMonthMoney(DebtSelGrantModel model) 位置 G:\NET项目\TaoCode\HCQ2\HCQ2WebAPI_Logic\APPController\DebtWageController.cs:行号 85
   在 System.Web.Http.Controllers.ReflectedHttpActionDescriptor.ActionExecutor.<>c__DisplayClass10.<GetExecutor>b__9(Object instance, Object[] methodParameters)
   在 System.Web.Http.Controllers.ReflectedHttpActionDescriptor.ExecuteAsync(HttpControllerContext controllerContext, IDictionary`2 arguments, CancellationToken cancellationToken)
--- 引发异常的上一位置中堆栈跟踪的末尾 ---
   在 System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   在 System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   在 System.Web.Http.Controllers.ApiControllerActionInvoker.<InvokeActionAsyncCore>d__0.MoveNext()
--- 引发异常的上一位置中堆栈跟踪的末尾 ---
   在 System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   在 System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   在 System.Web.Http.Controllers.ActionFilterResult.<ExecuteAsync>d__2.MoveNext()
--- 引发异常的上一位置中堆栈跟踪的末尾 ---
   在 System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   在 System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   在 System.Web.Http.Filters.AuthorizationFilterAttribute.<ExecuteAuthorizationFilterAsyncCore>d__2.MoveNext()
--- 引发异常的上一位置中堆栈跟踪的末尾 ---
   在 System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   在 System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   在 System.Web.Http.Controllers.AuthenticationFilterResult.<ExecuteAsync>d__0.MoveNext()
--- 引发异常的上一位置中堆栈跟踪的末尾 ---
   在 System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   在 System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   在 System.Web.Http.Controllers.ExceptionFilterResult.<ExecuteAsync>d__0.MoveNext()

 

 
记录时间：2017-05-19 17:02:13,487 
线程ID:[21] 
日志级别：  ERROR 
出错类：HCQ2WebAPI_Logic.DExceptionFilterAttribute property: [(null)] - 
错误描述：
.捕获异常信息：
IP：::1
浏览器：Unknown
浏览器版本：0
操作系统：Unknown
错误信息：
错误页面：http://localhost/HCQ2WebAPI/api/DebtWage/GetDebtMonthMoney
错误信息：从字符串转换日期和/或时间时，转换失败。
错误源：.Net SqlClient Data Provider
异常方法：Void OnError(System.Data.SqlClient.SqlException, Boolean, System.Action`1[System.Action])
堆栈信息：   在 System.Data.SqlClient.SqlConnection.OnError(SqlException exception, Boolean breakConnection, Action`1 wrapCloseInAction)
   在 System.Data.SqlClient.TdsParser.ThrowExceptionAndWarning(TdsParserStateObject stateObj, Boolean callerHasConnectionLock, Boolean asyncClose)
   在 System.Data.SqlClient.TdsParser.TryRun(RunBehavior runBehavior, SqlCommand cmdHandler, SqlDataReader dataStream, BulkCopySimpleResultSet bulkCopyHandler, TdsParserStateObject stateObj, Boolean& dataReady)
   在 System.Data.SqlClient.SqlDataReader.TryHasMoreRows(Boolean& moreRows)
   在 System.Data.SqlClient.SqlDataReader.TryReadInternal(Boolean setTimeout, Boolean& more)
   在 System.Data.SqlClient.SqlDataReader.Read()
   在 System.Data.Common.DataAdapter.FillLoadDataRow(SchemaMapping mapping)
   在 System.Data.Common.DataAdapter.FillFromReader(DataSet dataset, DataTable datatable, String srcTable, DataReaderContainer dataReader, Int32 startRecord, Int32 maxRecords, DataColumn parentChapterColumn, Object parentChapterValue)
   在 System.Data.Common.DataAdapter.Fill(DataTable[] dataTables, IDataReader dataReader, Int32 startRecord, Int32 maxRecords)
   在 System.Data.Common.DbDataAdapter.FillInternal(DataSet dataset, DataTable[] datatables, Int32 startRecord, Int32 maxRecords, String srcTable, IDbCommand command, CommandBehavior behavior)
   在 System.Data.Common.DbDataAdapter.Fill(DataTable[] dataTables, Int32 startRecord, Int32 maxRecords, IDbCommand command, CommandBehavior behavior)
   在 System.Data.Common.DbDataAdapter.Fill(DataTable dataTable)
   在 HCQ2_Common.SQL.SqlHelper.ExecuteDataTable(String sql, CommandType cmdType, SqlParameter[] psm) 位置 G:\NET项目\TaoCode\HCQ2\HCQ2_Common\SQL\SqlHelper.cs:行号 176
   在 HCQ2_DAL_MSSQL.View_QXTJDAL.GetDebtChartMonthDataByOrg(DebtSelGrantModel model) 位置 G:\NET项目\TaoCode\HCQ2\HCQ2_DAL\ExtensionDAL\View_QXTJDAL.cs:行号 201
   在 HCQ2_BLL.View_QXTJBLL.GetDebtChartMonthDataByOrg(DebtSelGrantModel model) 位置 G:\NET项目\TaoCode\HCQ2\HCQ2_BLL\ExtensionBLL\View_QXTJBLL.cs:行号 280
   在 HCQ2WebAPI_Logic.APPController.DebtWageController.GetDebtMonthMoney(DebtSelGrantModel model) 位置 G:\NET项目\TaoCode\HCQ2\HCQ2WebAPI_Logic\APPController\DebtWageController.cs:行号 85
   在 System.Web.Http.Controllers.ReflectedHttpActionDescriptor.ActionExecutor.<>c__DisplayClass10.<GetExecutor>b__9(Object instance, Object[] methodParameters)
   在 System.Web.Http.Controllers.ReflectedHttpActionDescriptor.ExecuteAsync(HttpControllerContext controllerContext, IDictionary`2 arguments, CancellationToken cancellationToken)
--- 引发异常的上一位置中堆栈跟踪的末尾 ---
   在 System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   在 System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   在 System.Web.Http.Controllers.ApiControllerActionInvoker.<InvokeActionAsyncCore>d__0.MoveNext()
--- 引发异常的上一位置中堆栈跟踪的末尾 ---
   在 System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   在 System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   在 System.Web.Http.Controllers.ActionFilterResult.<ExecuteAsync>d__2.MoveNext()
--- 引发异常的上一位置中堆栈跟踪的末尾 ---
   在 System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   在 System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   在 System.Web.Http.Filters.AuthorizationFilterAttribute.<ExecuteAuthorizationFilterAsyncCore>d__2.MoveNext()
--- 引发异常的上一位置中堆栈跟踪的末尾 ---
   在 System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   在 System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   在 System.Web.Http.Controllers.AuthenticationFilterResult.<ExecuteAsync>d__0.MoveNext()
--- 引发异常的上一位置中堆栈跟踪的末尾 ---
   在 System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   在 System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   在 System.Web.Http.Controllers.ExceptionFilterResult.<ExecuteAsync>d__0.MoveNext()

 

 
记录时间：2017-05-19 17:03:40,057 
线程ID:[22] 
日志级别：  ERROR 
出错类：HCQ2WebAPI_Logic.DExceptionFilterAttribute property: [(null)] - 
错误描述：
.捕获异常信息：
IP：::1
浏览器：Unknown
浏览器版本：0
操作系统：Unknown
错误信息：
错误页面：http://localhost/HCQ2WebAPI/api/DebtWage/GetDebtMonthMoney
错误信息：从字符串转换日期和/或时间时，转换失败。
错误源：.Net SqlClient Data Provider
异常方法：Void OnError(System.Data.SqlClient.SqlException, Boolean, System.Action`1[System.Action])
堆栈信息：   在 System.Data.SqlClient.SqlConnection.OnError(SqlException exception, Boolean breakConnection, Action`1 wrapCloseInAction)
   在 System.Data.SqlClient.TdsParser.ThrowExceptionAndWarning(TdsParserStateObject stateObj, Boolean callerHasConnectionLock, Boolean asyncClose)
   在 System.Data.SqlClient.TdsParser.TryRun(RunBehavior runBehavior, SqlCommand cmdHandler, SqlDataReader dataStream, BulkCopySimpleResultSet bulkCopyHandler, TdsParserStateObject stateObj, Boolean& dataReady)
   在 System.Data.SqlClient.SqlDataReader.TryHasMoreRows(Boolean& moreRows)
   在 System.Data.SqlClient.SqlDataReader.TryReadInternal(Boolean setTimeout, Boolean& more)
   在 System.Data.SqlClient.SqlDataReader.Read()
   在 System.Data.Common.DataAdapter.FillLoadDataRow(SchemaMapping mapping)
   在 System.Data.Common.DataAdapter.FillFromReader(DataSet dataset, DataTable datatable, String srcTable, DataReaderContainer dataReader, Int32 startRecord, Int32 maxRecords, DataColumn parentChapterColumn, Object parentChapterValue)
   在 System.Data.Common.DataAdapter.Fill(DataTable[] dataTables, IDataReader dataReader, Int32 startRecord, Int32 maxRecords)
   在 System.Data.Common.DbDataAdapter.FillInternal(DataSet dataset, DataTable[] datatables, Int32 startRecord, Int32 maxRecords, String srcTable, IDbCommand command, CommandBehavior behavior)
   在 System.Data.Common.DbDataAdapter.Fill(DataTable[] dataTables, Int32 startRecord, Int32 maxRecords, IDbCommand command, CommandBehavior behavior)
   在 System.Data.Common.DbDataAdapter.Fill(DataTable dataTable)
   在 HCQ2_Common.SQL.SqlHelper.ExecuteDataTable(String sql, CommandType cmdType, SqlParameter[] psm) 位置 G:\NET项目\TaoCode\HCQ2\HCQ2_Common\SQL\SqlHelper.cs:行号 176
   在 HCQ2_DAL_MSSQL.View_QXTJDAL.GetDebtChartMonthDataByOrg(DebtSelGrantModel model) 位置 G:\NET项目\TaoCode\HCQ2\HCQ2_DAL\ExtensionDAL\View_QXTJDAL.cs:行号 201
   在 HCQ2_BLL.View_QXTJBLL.GetDebtChartMonthDataByOrg(DebtSelGrantModel model) 位置 G:\NET项目\TaoCode\HCQ2\HCQ2_BLL\ExtensionBLL\View_QXTJBLL.cs:行号 280
   在 HCQ2WebAPI_Logic.APPController.DebtWageController.GetDebtMonthMoney(DebtSelGrantModel model) 位置 G:\NET项目\TaoCode\HCQ2\HCQ2WebAPI_Logic\APPController\DebtWageController.cs:行号 85
   在 System.Web.Http.Controllers.ReflectedHttpActionDescriptor.ActionExecutor.<>c__DisplayClass10.<GetExecutor>b__9(Object instance, Object[] methodParameters)
   在 System.Web.Http.Controllers.ReflectedHttpActionDescriptor.ExecuteAsync(HttpControllerContext controllerContext, IDictionary`2 arguments, CancellationToken cancellationToken)
--- 引发异常的上一位置中堆栈跟踪的末尾 ---
   在 System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   在 System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   在 System.Web.Http.Controllers.ApiControllerActionInvoker.<InvokeActionAsyncCore>d__0.MoveNext()
--- 引发异常的上一位置中堆栈跟踪的末尾 ---
   在 System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   在 System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   在 System.Web.Http.Controllers.ActionFilterResult.<ExecuteAsync>d__2.MoveNext()
--- 引发异常的上一位置中堆栈跟踪的末尾 ---
   在 System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   在 System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   在 System.Web.Http.Filters.AuthorizationFilterAttribute.<ExecuteAuthorizationFilterAsyncCore>d__2.MoveNext()
--- 引发异常的上一位置中堆栈跟踪的末尾 ---
   在 System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   在 System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   在 System.Web.Http.Controllers.AuthenticationFilterResult.<ExecuteAsync>d__0.MoveNext()
--- 引发异常的上一位置中堆栈跟踪的末尾 ---
   在 System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   在 System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   在 System.Web.Http.Controllers.ExceptionFilterResult.<ExecuteAsync>d__0.MoveNext()

 

 
记录时间：2017-05-19 17:10:49,889 
线程ID:[29] 
日志级别：  ERROR 
出错类：HCQ2WebAPI_Logic.DExceptionFilterAttribute property: [(null)] - 
错误描述：
.捕获异常信息：
IP：::1
浏览器：Unknown
浏览器版本：0
操作系统：Unknown
错误信息：
错误页面：http://localhost/HCQ2WebAPI/api/DebtWage/GetDebtMonthMoney
错误信息：从字符串转换日期和/或时间时，转换失败。
错误源：.Net SqlClient Data Provider
异常方法：Void OnError(System.Data.SqlClient.SqlException, Boolean, System.Action`1[System.Action])
堆栈信息：   在 System.Data.SqlClient.SqlConnection.OnError(SqlException exception, Boolean breakConnection, Action`1 wrapCloseInAction)
   在 System.Data.SqlClient.TdsParser.ThrowExceptionAndWarning(TdsParserStateObject stateObj, Boolean callerHasConnectionLock, Boolean asyncClose)
   在 System.Data.SqlClient.TdsParser.TryRun(RunBehavior runBehavior, SqlCommand cmdHandler, SqlDataReader dataStream, BulkCopySimpleResultSet bulkCopyHandler, TdsParserStateObject stateObj, Boolean& dataReady)
   在 System.Data.SqlClient.SqlDataReader.TryHasMoreRows(Boolean& moreRows)
   在 System.Data.SqlClient.SqlDataReader.TryReadInternal(Boolean setTimeout, Boolean& more)
   在 System.Data.SqlClient.SqlDataReader.Read()
   在 System.Data.Common.DataAdapter.FillLoadDataRow(SchemaMapping mapping)
   在 System.Data.Common.DataAdapter.FillFromReader(DataSet dataset, DataTable datatable, String srcTable, DataReaderContainer dataReader, Int32 startRecord, Int32 maxRecords, DataColumn parentChapterColumn, Object parentChapterValue)
   在 System.Data.Common.DataAdapter.Fill(DataTable[] dataTables, IDataReader dataReader, Int32 startRecord, Int32 maxRecords)
   在 System.Data.Common.DbDataAdapter.FillInternal(DataSet dataset, DataTable[] datatables, Int32 startRecord, Int32 maxRecords, String srcTable, IDbCommand command, CommandBehavior behavior)
   在 System.Data.Common.DbDataAdapter.Fill(DataTable[] dataTables, Int32 startRecord, Int32 maxRecords, IDbCommand command, CommandBehavior behavior)
   在 System.Data.Common.DbDataAdapter.Fill(DataTable dataTable)
   在 HCQ2_Common.SQL.SqlHelper.ExecuteDataTable(String sql, CommandType cmdType, SqlParameter[] psm) 位置 G:\NET项目\TaoCode\HCQ2\HCQ2_Common\SQL\SqlHelper.cs:行号 176
   在 HCQ2_DAL_MSSQL.View_QXTJDAL.GetDebtChartEndMonthDataByOrg(DebtSelGrantModel model) 位置 G:\NET项目\TaoCode\HCQ2\HCQ2_DAL\ExtensionDAL\View_QXTJDAL.cs:行号 234
   在 HCQ2_BLL.View_QXTJBLL.GetDebtChartEndMonthDataByOrg(DebtSelGrantModel model) 位置 G:\NET项目\TaoCode\HCQ2\HCQ2_BLL\ExtensionBLL\View_QXTJBLL.cs:行号 289
   在 HCQ2WebAPI_Logic.APPController.DebtWageController.GetDebtMonthMoney(DebtSelGrantModel model) 位置 G:\NET项目\TaoCode\HCQ2\HCQ2WebAPI_Logic\APPController\DebtWageController.cs:行号 86
   在 System.Web.Http.Controllers.ReflectedHttpActionDescriptor.ActionExecutor.<>c__DisplayClass10.<GetExecutor>b__9(Object instance, Object[] methodParameters)
   在 System.Web.Http.Controllers.ReflectedHttpActionDescriptor.ExecuteAsync(HttpControllerContext controllerContext, IDictionary`2 arguments, CancellationToken cancellationToken)
--- 引发异常的上一位置中堆栈跟踪的末尾 ---
   在 System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   在 System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   在 System.Web.Http.Controllers.ApiControllerActionInvoker.<InvokeActionAsyncCore>d__0.MoveNext()
--- 引发异常的上一位置中堆栈跟踪的末尾 ---
   在 System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   在 System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   在 System.Web.Http.Controllers.ActionFilterResult.<ExecuteAsync>d__2.MoveNext()
--- 引发异常的上一位置中堆栈跟踪的末尾 ---
   在 System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   在 System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   在 System.Web.Http.Filters.AuthorizationFilterAttribute.<ExecuteAuthorizationFilterAsyncCore>d__2.MoveNext()
--- 引发异常的上一位置中堆栈跟踪的末尾 ---
   在 System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   在 System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   在 System.Web.Http.Controllers.AuthenticationFilterResult.<ExecuteAsync>d__0.MoveNext()
--- 引发异常的上一位置中堆栈跟踪的末尾 ---
   在 System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   在 System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   在 System.Web.Http.Controllers.ExceptionFilterResult.<ExecuteAsync>d__0.MoveNext()

 

 
记录时间：2017-05-19 17:12:51,825 
线程ID:[38] 
日志级别：  ERROR 
出错类：HCQ2WebAPI_Logic.DExceptionFilterAttribute property: [(null)] - 
错误描述：
.捕获异常信息：
IP：::1
浏览器：Unknown
浏览器版本：0
操作系统：Unknown
错误信息：
错误页面：http://localhost/HCQ2WebAPI/api/DebtWage/GetDebtMonthMoney
错误信息：从字符串转换日期和/或时间时，转换失败。
错误源：.Net SqlClient Data Provider
异常方法：Void OnError(System.Data.SqlClient.SqlException, Boolean, System.Action`1[System.Action])
堆栈信息：   在 System.Data.SqlClient.SqlConnection.OnError(SqlException exception, Boolean breakConnection, Action`1 wrapCloseInAction)
   在 System.Data.SqlClient.TdsParser.ThrowExceptionAndWarning(TdsParserStateObject stateObj, Boolean callerHasConnectionLock, Boolean asyncClose)
   在 System.Data.SqlClient.TdsParser.TryRun(RunBehavior runBehavior, SqlCommand cmdHandler, SqlDataReader dataStream, BulkCopySimpleResultSet bulkCopyHandler, TdsParserStateObject stateObj, Boolean& dataReady)
   在 System.Data.SqlClient.SqlDataReader.TryHasMoreRows(Boolean& moreRows)
   在 System.Data.SqlClient.SqlDataReader.TryReadInternal(Boolean setTimeout, Boolean& more)
   在 System.Data.SqlClient.SqlDataReader.Read()
   在 System.Data.Common.DataAdapter.FillLoadDataRow(SchemaMapping mapping)
   在 System.Data.Common.DataAdapter.FillFromReader(DataSet dataset, DataTable datatable, String srcTable, DataReaderContainer dataReader, Int32 startRecord, Int32 maxRecords, DataColumn parentChapterColumn, Object parentChapterValue)
   在 System.Data.Common.DataAdapter.Fill(DataTable[] dataTables, IDataReader dataReader, Int32 startRecord, Int32 maxRecords)
   在 System.Data.Common.DbDataAdapter.FillInternal(DataSet dataset, DataTable[] datatables, Int32 startRecord, Int32 maxRecords, String srcTable, IDbCommand command, CommandBehavior behavior)
   在 System.Data.Common.DbDataAdapter.Fill(DataTable[] dataTables, Int32 startRecord, Int32 maxRecords, IDbCommand command, CommandBehavior behavior)
   在 System.Data.Common.DbDataAdapter.Fill(DataTable dataTable)
   在 HCQ2_Common.SQL.SqlHelper.ExecuteDataTable(String sql, CommandType cmdType, SqlParameter[] psm) 位置 G:\NET项目\TaoCode\HCQ2\HCQ2_Common\SQL\SqlHelper.cs:行号 176
   在 HCQ2_DAL_MSSQL.View_QXTJDAL.GetDebtChartEndMonthDataByOrg(DebtSelGrantModel model) 位置 G:\NET项目\TaoCode\HCQ2\HCQ2_DAL\ExtensionDAL\View_QXTJDAL.cs:行号 234
   在 HCQ2_BLL.View_QXTJBLL.GetDebtChartEndMonthDataByOrg(DebtSelGrantModel model) 位置 G:\NET项目\TaoCode\HCQ2\HCQ2_BLL\ExtensionBLL\View_QXTJBLL.cs:行号 289
   在 HCQ2WebAPI_Logic.APPController.DebtWageController.GetDebtMonthMoney(DebtSelGrantModel model) 位置 G:\NET项目\TaoCode\HCQ2\HCQ2WebAPI_Logic\APPController\DebtWageController.cs:行号 86
   在 System.Web.Http.Controllers.ReflectedHttpActionDescriptor.ActionExecutor.<>c__DisplayClass10.<GetExecutor>b__9(Object instance, Object[] methodParameters)
   在 System.Web.Http.Controllers.ReflectedHttpActionDescriptor.ExecuteAsync(HttpControllerContext controllerContext, IDictionary`2 arguments, CancellationToken cancellationToken)
--- 引发异常的上一位置中堆栈跟踪的末尾 ---
   在 System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   在 System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   在 System.Web.Http.Controllers.ApiControllerActionInvoker.<InvokeActionAsyncCore>d__0.MoveNext()
--- 引发异常的上一位置中堆栈跟踪的末尾 ---
   在 System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   在 System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   在 System.Web.Http.Controllers.ActionFilterResult.<ExecuteAsync>d__2.MoveNext()
--- 引发异常的上一位置中堆栈跟踪的末尾 ---
   在 System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   在 System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   在 System.Web.Http.Filters.AuthorizationFilterAttribute.<ExecuteAuthorizationFilterAsyncCore>d__2.MoveNext()
--- 引发异常的上一位置中堆栈跟踪的末尾 ---
   在 System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   在 System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   在 System.Web.Http.Controllers.AuthenticationFilterResult.<ExecuteAsync>d__0.MoveNext()
--- 引发异常的上一位置中堆栈跟踪的末尾 ---
   在 System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   在 System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   在 System.Web.Http.Controllers.ExceptionFilterResult.<ExecuteAsync>d__0.MoveNext()

 

 
记录时间：2017-05-19 17:14:59,024 
线程ID:[24] 
日志级别：  ERROR 
出错类：HCQ2WebAPI_Logic.DExceptionFilterAttribute property: [(null)] - 
错误描述：
.捕获异常信息：
IP：::1
浏览器：Unknown
浏览器版本：0
操作系统：Unknown
错误信息：
错误页面：http://localhost/HCQ2WebAPI/api/DebtWage/GetDebtMonthMoney
错误信息：从字符串转换日期和/或时间时，转换失败。
错误源：.Net SqlClient Data Provider
异常方法：Void OnError(System.Data.SqlClient.SqlException, Boolean, System.Action`1[System.Action])
堆栈信息：   在 System.Data.SqlClient.SqlConnection.OnError(SqlException exception, Boolean breakConnection, Action`1 wrapCloseInAction)
   在 System.Data.SqlClient.TdsParser.ThrowExceptionAndWarning(TdsParserStateObject stateObj, Boolean callerHasConnectionLock, Boolean asyncClose)
   在 System.Data.SqlClient.TdsParser.TryRun(RunBehavior runBehavior, SqlCommand cmdHandler, SqlDataReader dataStream, BulkCopySimpleResultSet bulkCopyHandler, TdsParserStateObject stateObj, Boolean& dataReady)
   在 System.Data.SqlClient.SqlDataReader.TryHasMoreRows(Boolean& moreRows)
   在 System.Data.SqlClient.SqlDataReader.TryReadInternal(Boolean setTimeout, Boolean& more)
   在 System.Data.SqlClient.SqlDataReader.Read()
   在 System.Data.Common.DataAdapter.FillLoadDataRow(SchemaMapping mapping)
   在 System.Data.Common.DataAdapter.FillFromReader(DataSet dataset, DataTable datatable, String srcTable, DataReaderContainer dataReader, Int32 startRecord, Int32 maxRecords, DataColumn parentChapterColumn, Object parentChapterValue)
   在 System.Data.Common.DataAdapter.Fill(DataTable[] dataTables, IDataReader dataReader, Int32 startRecord, Int32 maxRecords)
   在 System.Data.Common.DbDataAdapter.FillInternal(DataSet dataset, DataTable[] datatables, Int32 startRecord, Int32 maxRecords, String srcTable, IDbCommand command, CommandBehavior behavior)
   在 System.Data.Common.DbDataAdapter.Fill(DataTable[] dataTables, Int32 startRecord, Int32 maxRecords, IDbCommand command, CommandBehavior behavior)
   在 System.Data.Common.DbDataAdapter.Fill(DataTable dataTable)
   在 HCQ2_Common.SQL.SqlHelper.ExecuteDataTable(String sql, CommandType cmdType, SqlParameter[] psm) 位置 G:\NET项目\TaoCode\HCQ2\HCQ2_Common\SQL\SqlHelper.cs:行号 176
   在 HCQ2_DAL_MSSQL.View_QXTJDAL.GetDebtChartEndMonthDataByOrg(DebtSelGrantModel model) 位置 G:\NET项目\TaoCode\HCQ2\HCQ2_DAL\ExtensionDAL\View_QXTJDAL.cs:行号 234
   在 HCQ2_BLL.View_QXTJBLL.GetDebtChartEndMonthDataByOrg(DebtSelGrantModel model) 位置 G:\NET项目\TaoCode\HCQ2\HCQ2_BLL\ExtensionBLL\View_QXTJBLL.cs:行号 289
   在 HCQ2WebAPI_Logic.APPController.DebtWageController.GetDebtMonthMoney(DebtSelGrantModel model) 位置 G:\NET项目\TaoCode\HCQ2\HCQ2WebAPI_Logic\APPController\DebtWageController.cs:行号 86
   在 System.Web.Http.Controllers.ReflectedHttpActionDescriptor.ActionExecutor.<>c__DisplayClass10.<GetExecutor>b__9(Object instance, Object[] methodParameters)
   在 System.Web.Http.Controllers.ReflectedHttpActionDescriptor.ExecuteAsync(HttpControllerContext controllerContext, IDictionary`2 arguments, CancellationToken cancellationToken)
--- 引发异常的上一位置中堆栈跟踪的末尾 ---
   在 System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   在 System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   在 System.Web.Http.Controllers.ApiControllerActionInvoker.<InvokeActionAsyncCore>d__0.MoveNext()
--- 引发异常的上一位置中堆栈跟踪的末尾 ---
   在 System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   在 System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   在 System.Web.Http.Controllers.ActionFilterResult.<ExecuteAsync>d__2.MoveNext()
--- 引发异常的上一位置中堆栈跟踪的末尾 ---
   在 System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   在 System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   在 System.Web.Http.Filters.AuthorizationFilterAttribute.<ExecuteAuthorizationFilterAsyncCore>d__2.MoveNext()
--- 引发异常的上一位置中堆栈跟踪的末尾 ---
   在 System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   在 System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   在 System.Web.Http.Controllers.AuthenticationFilterResult.<ExecuteAsync>d__0.MoveNext()
--- 引发异常的上一位置中堆栈跟踪的末尾 ---
   在 System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   在 System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   在 System.Web.Http.Controllers.ExceptionFilterResult.<ExecuteAsync>d__0.MoveNext()

 

 
记录时间：2017-05-19 17:15:29,211 
线程ID:[38] 
日志级别：  ERROR 
出错类：HCQ2WebAPI_Logic.DExceptionFilterAttribute property: [(null)] - 
错误描述：
.捕获异常信息：
IP：::1
浏览器：Unknown
浏览器版本：0
操作系统：Unknown
错误信息：
错误页面：http://localhost/HCQ2WebAPI/api/DebtWage/GetDebtMonthMoney
错误信息：从字符串转换日期和/或时间时，转换失败。
错误源：.Net SqlClient Data Provider
异常方法：Void OnError(System.Data.SqlClient.SqlException, Boolean, System.Action`1[System.Action])
堆栈信息：   在 System.Data.SqlClient.SqlConnection.OnError(SqlException exception, Boolean breakConnection, Action`1 wrapCloseInAction)
   在 System.Data.SqlClient.TdsParser.ThrowExceptionAndWarning(TdsParserStateObject stateObj, Boolean callerHasConnectionLock, Boolean asyncClose)
   在 System.Data.SqlClient.TdsParser.TryRun(RunBehavior runBehavior, SqlCommand cmdHandler, SqlDataReader dataStream, BulkCopySimpleResultSet bulkCopyHandler, TdsParserStateObject stateObj, Boolean& dataReady)
   在 System.Data.SqlClient.SqlDataReader.TryHasMoreRows(Boolean& moreRows)
   在 System.Data.SqlClient.SqlDataReader.TryReadInternal(Boolean setTimeout, Boolean& more)
   在 System.Data.SqlClient.SqlDataReader.Read()
   在 System.Data.Common.DataAdapter.FillLoadDataRow(SchemaMapping mapping)
   在 System.Data.Common.DataAdapter.FillFromReader(DataSet dataset, DataTable datatable, String srcTable, DataReaderContainer dataReader, Int32 startRecord, Int32 maxRecords, DataColumn parentChapterColumn, Object parentChapterValue)
   在 System.Data.Common.DataAdapter.Fill(DataTable[] dataTables, IDataReader dataReader, Int32 startRecord, Int32 maxRecords)
   在 System.Data.Common.DbDataAdapter.FillInternal(DataSet dataset, DataTable[] datatables, Int32 startRecord, Int32 maxRecords, String srcTable, IDbCommand command, CommandBehavior behavior)
   在 System.Data.Common.DbDataAdapter.Fill(DataTable[] dataTables, Int32 startRecord, Int32 maxRecords, IDbCommand command, CommandBehavior behavior)
   在 System.Data.Common.DbDataAdapter.Fill(DataTable dataTable)
   在 HCQ2_Common.SQL.SqlHelper.ExecuteDataTable(String sql, CommandType cmdType, SqlParameter[] psm) 位置 G:\NET项目\TaoCode\HCQ2\HCQ2_Common\SQL\SqlHelper.cs:行号 176
   在 HCQ2_DAL_MSSQL.View_QXTJDAL.GetDebtChartEndMonthDataByOrg(DebtSelGrantModel model) 位置 G:\NET项目\TaoCode\HCQ2\HCQ2_DAL\ExtensionDAL\View_QXTJDAL.cs:行号 234
   在 HCQ2_BLL.View_QXTJBLL.GetDebtChartEndMonthDataByOrg(DebtSelGrantModel model) 位置 G:\NET项目\TaoCode\HCQ2\HCQ2_BLL\ExtensionBLL\View_QXTJBLL.cs:行号 289
   在 HCQ2WebAPI_Logic.APPController.DebtWageController.GetDebtMonthMoney(DebtSelGrantModel model) 位置 G:\NET项目\TaoCode\HCQ2\HCQ2WebAPI_Logic\APPController\DebtWageController.cs:行号 86
   在 System.Web.Http.Controllers.ReflectedHttpActionDescriptor.ActionExecutor.<>c__DisplayClass10.<GetExecutor>b__9(Object instance, Object[] methodParameters)
   在 System.Web.Http.Controllers.ReflectedHttpActionDescriptor.ExecuteAsync(HttpControllerContext controllerContext, IDictionary`2 arguments, CancellationToken cancellationToken)
--- 引发异常的上一位置中堆栈跟踪的末尾 ---
   在 System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   在 System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   在 System.Web.Http.Controllers.ApiControllerActionInvoker.<InvokeActionAsyncCore>d__0.MoveNext()
--- 引发异常的上一位置中堆栈跟踪的末尾 ---
   在 System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   在 System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   在 System.Web.Http.Controllers.ActionFilterResult.<ExecuteAsync>d__2.MoveNext()
--- 引发异常的上一位置中堆栈跟踪的末尾 ---
   在 System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   在 System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   在 System.Web.Http.Filters.AuthorizationFilterAttribute.<ExecuteAuthorizationFilterAsyncCore>d__2.MoveNext()
--- 引发异常的上一位置中堆栈跟踪的末尾 ---
   在 System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   在 System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   在 System.Web.Http.Controllers.AuthenticationFilterResult.<ExecuteAsync>d__0.MoveNext()
--- 引发异常的上一位置中堆栈跟踪的末尾 ---
   在 System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   在 System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   在 System.Web.Http.Controllers.ExceptionFilterResult.<ExecuteAsync>d__0.MoveNext()

 

 
记录时间：2017-05-19 17:17:44,410 
线程ID:[24] 
日志级别：  ERROR 
出错类：HCQ2WebAPI_Logic.DExceptionFilterAttribute property: [(null)] - 
错误描述：
.捕获异常信息：
IP：::1
浏览器：Unknown
浏览器版本：0
操作系统：Unknown
错误信息：
错误页面：http://localhost/HCQ2WebAPI/api/DebtWage/GetDebtMonthMoney
错误信息：从字符串转换日期和/或时间时，转换失败。
错误源：.Net SqlClient Data Provider
异常方法：Void OnError(System.Data.SqlClient.SqlException, Boolean, System.Action`1[System.Action])
堆栈信息：   在 System.Data.SqlClient.SqlConnection.OnError(SqlException exception, Boolean breakConnection, Action`1 wrapCloseInAction)
   在 System.Data.SqlClient.TdsParser.ThrowExceptionAndWarning(TdsParserStateObject stateObj, Boolean callerHasConnectionLock, Boolean asyncClose)
   在 System.Data.SqlClient.TdsParser.TryRun(RunBehavior runBehavior, SqlCommand cmdHandler, SqlDataReader dataStream, BulkCopySimpleResultSet bulkCopyHandler, TdsParserStateObject stateObj, Boolean& dataReady)
   在 System.Data.SqlClient.SqlDataReader.TryHasMoreRows(Boolean& moreRows)
   在 System.Data.SqlClient.SqlDataReader.TryReadInternal(Boolean setTimeout, Boolean& more)
   在 System.Data.SqlClient.SqlDataReader.Read()
   在 System.Data.Common.DataAdapter.FillLoadDataRow(SchemaMapping mapping)
   在 System.Data.Common.DataAdapter.FillFromReader(DataSet dataset, DataTable datatable, String srcTable, DataReaderContainer dataReader, Int32 startRecord, Int32 maxRecords, DataColumn parentChapterColumn, Object parentChapterValue)
   在 System.Data.Common.DataAdapter.Fill(DataTable[] dataTables, IDataReader dataReader, Int32 startRecord, Int32 maxRecords)
   在 System.Data.Common.DbDataAdapter.FillInternal(DataSet dataset, DataTable[] datatables, Int32 startRecord, Int32 maxRecords, String srcTable, IDbCommand command, CommandBehavior behavior)
   在 System.Data.Common.DbDataAdapter.Fill(DataTable[] dataTables, Int32 startRecord, Int32 maxRecords, IDbCommand command, CommandBehavior behavior)
   在 System.Data.Common.DbDataAdapter.Fill(DataTable dataTable)
   在 HCQ2_Common.SQL.SqlHelper.ExecuteDataTable(String sql, CommandType cmdType, SqlParameter[] psm) 位置 G:\NET项目\TaoCode\HCQ2\HCQ2_Common\SQL\SqlHelper.cs:行号 176
   在 HCQ2_DAL_MSSQL.View_QXTJDAL.GetDebtChartEndMonthDataByOrg(DebtSelGrantModel model) 位置 G:\NET项目\TaoCode\HCQ2\HCQ2_DAL\ExtensionDAL\View_QXTJDAL.cs:行号 235
   在 HCQ2_BLL.View_QXTJBLL.GetDebtChartEndMonthDataByOrg(DebtSelGrantModel model) 位置 G:\NET项目\TaoCode\HCQ2\HCQ2_BLL\ExtensionBLL\View_QXTJBLL.cs:行号 289
   在 HCQ2WebAPI_Logic.APPController.DebtWageController.GetDebtMonthMoney(DebtSelGrantModel model) 位置 G:\NET项目\TaoCode\HCQ2\HCQ2WebAPI_Logic\APPController\DebtWageController.cs:行号 86
   在 System.Web.Http.Controllers.ReflectedHttpActionDescriptor.ActionExecutor.<>c__DisplayClass10.<GetExecutor>b__9(Object instance, Object[] methodParameters)
   在 System.Web.Http.Controllers.ReflectedHttpActionDescriptor.ExecuteAsync(HttpControllerContext controllerContext, IDictionary`2 arguments, CancellationToken cancellationToken)
--- 引发异常的上一位置中堆栈跟踪的末尾 ---
   在 System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   在 System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   在 System.Web.Http.Controllers.ApiControllerActionInvoker.<InvokeActionAsyncCore>d__0.MoveNext()
--- 引发异常的上一位置中堆栈跟踪的末尾 ---
   在 System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   在 System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   在 System.Web.Http.Controllers.ActionFilterResult.<ExecuteAsync>d__2.MoveNext()
--- 引发异常的上一位置中堆栈跟踪的末尾 ---
   在 System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   在 System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   在 System.Web.Http.Filters.AuthorizationFilterAttribute.<ExecuteAuthorizationFilterAsyncCore>d__2.MoveNext()
--- 引发异常的上一位置中堆栈跟踪的末尾 ---
   在 System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   在 System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   在 System.Web.Http.Controllers.AuthenticationFilterResult.<ExecuteAsync>d__0.MoveNext()
--- 引发异常的上一位置中堆栈跟踪的末尾 ---
   在 System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   在 System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   在 System.Web.Http.Controllers.ExceptionFilterResult.<ExecuteAsync>d__0.MoveNext()

 
